packages/terra-framework-docs/src/terra-dev-site/test/menu/menu/CustomIconsMenu.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Menu from 'terra-menu';
+import IconWarning from 'terra-icon/lib/icon/IconWarning';
+import IconError from 'terra-icon/lib/icon/IconError';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconHelp from 'terra-icon/lib/icon/IconHelp';
+import IconRequired from 'terra-icon/lib/icon/IconRequired';
+
+class CustomIconsMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
```

Comment:
```suggestion
    this.forceUpdate();
```
why do we need forceUpdate on component mount ??

Reply:
was a copy and paste from another test, removed:
https://github.com/cerner/terra-framework/pull/1592/commits/3fc0c686f4747389c60a7d563883b80ed4e799e6